src/styles/__tests__/styled.test.js

```diff
@@ -42,6 +42,13 @@ test('styled', () => {
   wrapper.unmount();
 });
 
+test('styled can be called with single string argument', () => {
+  const ADiv = styled('div');
```

Comment:
Is this pattern useful? It's not something that [styletron-react](https://github.com/styletron/styletron/blob/master/packages/styletron-react/src/types.js#L52-L63) supports either.

I believe that failing to flow check in this case promotes simpler practices. It can be replaced with a `<div>` tag in this case, rather than calling to styletron.

Reply:
It may not be useful but it's valid and we have existing code that uses this. If we don't want to support this, styletron should error instead.

Reply:
sgtm - i'm going to consider adding a warning in this case to suggest a simpler approach